docs/docs/configuration/overview.md

```diff
@@ -26,6 +26,7 @@ An example [oauth2-proxy.cfg](https://github.com/oauth2-proxy/oauth2-proxy/blob/
 | `--authenticated-emails-file` | string | authenticate against emails via file (one per line) | |
 | `--azure-tenant` | string | go to a tenant-specific or common (tenant-independent) endpoint. | `"common"` |
 | `--basic-auth-password` | string | the password to set when passing the HTTP Basic Auth header | |
+| `--basic-auth-session-group` | string \| list | the groups to be set on all basic auth sessions | |
```

Comment:
Should this flag naming potentially align with `--htpasswd-file` -- since that is what controls getting username/password auth enabled?

Seems slightly misleading aligning here with `--basic-auth-password` since that deals with setting arbitrary user details on the authorization header to the backend.

Reply:
I thought the same thing here but was not sure, I had gone with --basic-auth-xx as I was inside the basic auth middleware. But tying it to the htpasswd does make sense.

I have made this `--htpasswd-user-group` LMK what you think.